src/ja/kadocomi/src/eu/kanade/tachiyomi/extension/ja/kadocomi/KadoComiDto.kt

```diff
@@ -0,0 +1,71 @@
+package eu.kanade.tachiyomi.extension.ja.kadocomi
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class KadoComiWorkDto(
```

Comment:
```suggestion
class KadoComiWorkDto(
```

Remove the `data` annotation because it generates methods that aren’t being used